### ISSUE TITLE
Looks like quotes in track titles cause issues

### ISSUE THREAD

**Issue by drobinson** [jakiestfu/off-the-rip#7]
This file _is_ in this directory, but looks like it may be having issues seeing it since there are double quotes (and spaces) in the track title.  Output pasted below:

$ otr dl https://soundcloud.com/artist/tracktitle_withspaces_and_quotes ~/Downloads
LOG: Sniffing network...
LOG: Downloading Artwork...
LOG: Resolving MP3 URI...
LOG: Downloading MP3...
LOG: Writing ID3 Tags...
eyeD3 --remove-all "/Users/d_rbn/Downloads/tracktitle withspaces "and with quotes".mp3"

Error: Command failed: /bin/sh -c env eyeD3 --remove-all "/Users/d_rbn/Downloads/tracktitle withspaces "and with quotes".mp3"
file not found: /Users/d_rbn/Downloads/tracktitle withspaces and

```
at ChildProcess.exithandler (child_process.js:751:12)
at ChildProcess.emit (events.js:110:17)
at maybeClose (child_process.js:1015:16)
at Process.ChildProcess._handle.onexit (child_process.js:1087:5)
```

LOG: Downloaded "tracktitle withspaces "and with quotes"" to /Users/d_rbn/Downloads/tracktitle withspaces "and with quotes".mp3

**Comment by jakiestfu** [comment]
Thanks, I'll get on this.

**Comment by drobinson** [comment]
removed track title and artist from issue as not to infringe on copyright - let me know if you need me to send the specific URL I used.

**Comment by jakiestfu** [comment]
I got it :)

**Comment by jakiestfu** [comment]
Fixed in `0.3.2`, run `npm update -g off-the-rip`

**Comment by drobinson** [comment]
![CONFIRMED](http://www.reactiongifs.us/wp-content/uploads/2013/08/picard_clapping.gif)
